### DISCUSSION TITLE
Bug with renaming yourself back

### DISCUSSION THREAD

**Discussion by munrocket** [community/community#23433] [New to GitHub]
<p>Step to reproduce:</p>
<ol>
<li>create an account</li>
<li>create a public repo</li>
<li>change username (and get some automatic redirection)</li>
<li>try to change it back and get an error</li>
</ol>
<pre><code class="lang-auto">Username change was not successful. The repository name `username1/myrepo` has been retired and cannot be reused
</code></pre>
<p>I read documentation about renaming and can’t see that you not allow to rename in back. I think that this is a bug with redirection. But if it’s not here PR in documentation</p><aside class="onebox githubpullrequest">
  <header class="source">

      <a href="https://github.com/github/docs/pull/7264" target="_blank" rel="noopener nofollow ugc">github.com/github/docs</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Pull Request">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 12 16" aria-hidden="true"><path d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/github/docs/pull/7264" target="_blank" rel="noopener nofollow ugc">Documentation about current bug on github</a>
    </h4>

    <div class="branches">
      <code>github:main</code> ← <code>munsocket:patch-1</code>
    </div>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-06-10" data-time="11:23:30" data-timezone="UTC">11:23AM - 10 Jun 21 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/munsocket" target="_blank" rel="noopener nofollow ugc">
          <img alt="munsocket" src="https://user-images.githubusercontent.com/20033246/181076894-91ee80b6-d53b-4eec-adf2-05fc000d8e82.png" class="onebox-avatar-inline" width="20" height="20">
          munsocket
        </a>
      </div>

      <div class="lines" title="1 commits changed 1 files with 1 additions and 1 deletions">
        <a href="https://github.com/github/docs/pull/7264/files" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+1</span>
          <span class="removed">-1</span>
        </a>
      </div>
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">Currently you can't rename yourself back.

Step to reproduce:
1) create an ac<span class="show-more-container"><a href="https://github.com/github/docs/pull/7264" target="_blank" rel="noopener nofollow ugc" class="show-more">…</a></span><span class="excerpt hidden">count
2) create a public repo
3) change username and get some automatic redirection
4) try to change it back and get an error
```
Username change was not successful. The repository name `username1/myrepo` has been retired and cannot be reused
```
I have a ticket with this bug in contact support #1180808 but if this is expected behavior, this should be in documentation.

&lt;!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
--&gt;

### Why:

Closes [issue link]

&lt;!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. --&gt;

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")</span></p>
  </div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>P.S. Yes I have a ticket <span class="hashtag">#1180808</span> with this bug in github support but they not answer. Anyone can take my previous name but I am not.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I’m afraid that one is going to have to be handled by support. I know it’s been a few days, but it’s in the correct queue and will be handled in order.</p>

**Comment by munrocket** [comment]
<p><span class="hashtag">#1167406</span> <strong>closed</strong> created 16 days ago<br>
<span class="hashtag">#1180808</span> <strong>open</strong> created 6 days ago</p>
<p>I am closed first ticket because nobody answer in 10 days and created new one 6 day ago. Is there any chance that GitHub Support didn’t see people with new username? But anyway I simply can’t rename myself back but… anybody can do it and take it because login is free.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p><a class="mention" href="/munsocket">@munsocket</a> if you close the ticket and open a new one, you go to the back of the queue. I am afraid that username tickets may be quite a wait, but closing a ticket isn’t going to help.</p>

**Comment by munrocket** [comment]
<p>Thanks, problem solved!</p>
